src/Arcus.WebApi.Tests.Integration/Logging/RequestTrackingMiddlewareTests.cs

```diff
@@ -1064,21 +1115,34 @@ private static bool ContainsLoggedEventContext(InMemorySink testSink)
 
         private static IDictionary<string, string> GetLoggedEventContext(InMemorySink testSink)
         {
-            IEnumerable<KeyValuePair<string, LogEventPropertyValue>> properties = 
+            IEnumerable<KeyValuePair<string, LogEventPropertyValue>> properties =
                 testSink.DequeueLogEvents()
                         .SelectMany(ev => ev.Properties);
 
             var logEntries = properties.Where(prop => prop.Key == ContextProperties.RequestTracking.RequestLogEntry);
             (string key, LogEventPropertyValue logEntry) = Assert.Single(logEntries);
             var requestLogEntry = Assert.IsType<StructureValue>(logEntry);
-            
-            LogEventProperty eventContext = 
+
+            LogEventProperty eventContext =
                 Assert.Single(requestLogEntry.Properties, property => property.Name == ContextProperties.TelemetryContext);
             var dictionaryValue = Assert.IsType<DictionaryValue>(eventContext.Value);
 
             return dictionaryValue.Elements.ToDictionary(
-                item => item.Key.ToStringValue(), 
+                item => item.Key.ToStringValue(),
                 item => item.Value.ToStringValue().Trim('\\', '\"', '[', ']'));
         }
+
+        private static IDictionary<string, string> GetRequestLogProperties(InMemorySink testSink)
+        {
+            IEnumerable<KeyValuePair<string, LogEventPropertyValue>> properties =
+                testSink.DequeueLogEvents()
+                    .SelectMany(ev => ev.Properties);
+
+            var logEntries = properties.Where(prop => prop.Key == ContextProperties.RequestTracking.RequestLogEntry);
+            (string key, LogEventPropertyValue logEntry) = Assert.Single(logEntries);
+            var requestLogEntry = Assert.IsType<StructureValue>(logEntry);
```

Comment:
This could be reused in other methods too, right?

Reply:
What exactly do you mean ?  It is made as a method, so it can be used in other test - methods ?

Reply:
Yeah, I see some duplication in the diff, so I thought that this method could be used elsewhere here as well. But could also be the code formatting that messed up the diff.

Reply:
This part is duplicated thrice now:
```csharp
 IEnumerable<KeyValuePair<string, LogEventPropertyValue>> properties =
                testSink.DequeueLogEvents()
                        .SelectMany(ev => ev.Properties);

 var eventContexts = properties.Where(prop => prop.Key == ContextProperties.RequestTracking.RequestLogEntry);
```
Which we can extract in a method.

Reply:
I'll have a look and see if it can be improved.